Triangulation_2/include/CGAL/Constrained_triangulation_2.h

```diff
@@ -40,16 +40,25 @@ struct Exact_predicates_tag{}; // to be used with filtered exact number
 
 
 template < class Gt, 
-           class Tds = Triangulation_data_structure_2 <
-                       Triangulation_vertex_base_2<Gt>,
-		       Constrained_triangulation_face_base_2<Gt> >, 
-           class Itag = No_intersection_tag >
-class Constrained_triangulation_2  : public Triangulation_2<Gt,Tds>
+           class Tds_ = Triangulation_data_structure_2 <
+                                                       Triangulation_vertex_base_2<Gt>,
+                                                       Constrained_triangulation_face_base_2<Gt> >,
+           class Itag_ = No_intersection_tag>
+class Constrained_triangulation_2  
+  : public Triangulation_2<Gt,typename Default::Get< Tds_,
+                                                     Triangulation_data_structure_2 <
+                                                       Triangulation_vertex_base_2<Gt>,
+                                                       Constrained_triangulation_face_base_2<Gt> > >::type >
 {
 
 public:
+  typedef typename Default::Get<Tds_, Triangulation_data_structure_2 <
+                                        Triangulation_vertex_base_2<Gt>,
+                                        Constrained_triangulation_face_base_2<Gt> > >::type Tds;
+  typedef typename Default::Get<Itag_, No_intersection_tag>::type Itag;
+
   typedef Triangulation_2<Gt,Tds> Triangulation;
-  typedef Constrained_triangulation_2<Gt,Tds,Itag>  Constrained_triangulation;
+  typedef Constrained_triangulation_2<Gt,Tds_,Itag>  Constrained_triangulation;
```

Comment:
I wasn't asking why you renamed the template parameter, I was asking specifically about line 61 why you forgot to update `Itag` to its new name `Itag_` in the typedef for `Constrained_triangulation`.